sssom/cli.py

```diff
@@ -89,6 +91,32 @@
     help="Fields.",
 )
 
+predicate_filter_option = click.option(
+    "-F",
+    "--mapping-predicate-filter",
+    multiple=True,
+    help="A list of predicates or a file path containing the list of predicates to be considered.",
+)
+
+
+def _get_list_of_predicate_iri(predicate_filter: tuple) -> list:
+    pred_filter_list = list(predicate_filter)
+    preds = [p for p in pred_filter_list if ":" in p]
+    preds_iri = [get_iri(p) for p in preds]
+
+    if len(preds) != len(pred_filter_list) and len(preds) > 0:
+        # The user passed file paths too.
+        pred_fps = [p for p in pred_filter_list if p not in preds]
+        if all(os.path.isfile(p) for p in pred_fps):
+            pred_list = list(
```

Comment:
we should get a warning if a user supplied option is neither a valid curie, nor a valid file path

Reply:
Done